meiki_ui/src/components/app/userForm/CreateUserForm.svelte

```diff
@@ -14,9 +13,14 @@
 
     async function onClick() {
         try {
-            await createAccount(userForm.getUsername(), userForm.getPassword())
+            await createUser(userForm.getUsername(), userForm.getPassword())
             dispatch("userCreated")
         } catch (err) {
+            if (err instanceof StatusNotOkError) {
+                error = await err.message
```

Comment:
Not sure if throwing the server error messages in the console is generally accepted or not (unless we're in debug mode or something). Could be a potential security issue?

Reply:
Ref: https://stackoverflow.com/a/67290755/5516481

_So the most ideal thing to do would be_

1. Have errors that can be detected by the frontend thrown right away without going to the server (like username should not have special chars, password should have atleast 5 characters). This reduces the load on the server and also better for UX because there is no lag in showing the error message
2. Errors that can only be detected by the backend (like username already exists) should come from the backend in a parsable format, like having error codes. Then the UI can parse the error message, have a switch case for each possible error code. This helps in supporting different languages and also allows us to show less verbose messages for UI users and more verbose messages for API users

_Security_

We should make sure our server messages does not reveal any sensitive information but that is true even without having an UI (We want to expose the API itself too in the future?). So I don't see any security issues as long as we are careful with our error messages

_So right now by doing what we are doing now we are loosing_ 

1. Ability to support multiple languages, 
2. Better UX  (somewhat? but our server does not lag or have any load anyways)

Reply:
Makes sense to block at the UI - That's definitely a todo. Agreed with the API being public part. I think what we have right now is in the right direction! Let's go for it and close the PRs!